context/app/api/client.py

```diff
@@ -121,49 +121,14 @@ def get_entity(self, uuid):
             raise Exception(f'UUID not unique; got {len(hits)} matches')
         entity = hits[0]['_source']
 
+        if 'metadata' not in entity:
+            entity['metadata'] = {}
```

Comment:
I think John might have done handled this case on the JSX side, too. Should this go away?

Reply:
I think so.

Reply:
removed.